### DISCUSSION TITLE
Self-hosted instance has 0.98 views per visit

### DISCUSSION THREAD

**Discussion by maximilianschmitt** [plausible/analytics#3935] [Self-Hosted Support]
I switched to the self-hosted version of Plausible a while ago and I've noticed that something is off about the way visitors are tracked:

- My views per visit are at **0.98** (previously a positive number)
- Bounce rate at **97%** (previously around 70%)
- Visit duration at **5s** (previously around 2min)

Is anybody else experiencing these issues or has a tip for how to debug this?

Thanks!

**Comment by rafalzawadzki** [comment]
I have exactly the same problem! On cloud, bounce rate was ~65%, self-hosted is 98% :/ same with time spent (couple of seconds). @maximilianschmitt were you able to solve this?

**Reply by maximilianschmitt** [reply]
Interesting! Are you also hosting with Dokku? I haven't been able to solve the problem unfortunately

**Reply by rafalzawadzki** [reply]
I’m hosting with Coolify, their one click deploy. 
Are you also using `next-plausible` by any chance?

**Reply by maximilianschmitt** [reply]
No, I was using the script directly. In Next.js as well (pages router) and using [rewrite rules](https://plausible.io/docs/proxy/guides/nextjs#using-raw-nextjs-config).

**Reply by rafalzawadzki** [reply]
Ok, so if I understand correctly you're also reverse-proxying like me. I wonder if this may be a clue... I checked the network calls (eg. `/api/event`) and they seem in order.

**Reply by maximilianschmitt** [reply]
Yep, I was also reverse-proxying. I also noticed that the self-hosted version does not recognize referrers as well as the hosted instance. I wonder if that's related in any way.

**Reply by rafalzawadzki** [reply]
Out of curiosity I added another project to my self-hosted Plausible and to my surprise, all metrics seem correct! Totally valid time spent, bounce rate at 60% etc.

I'm yet to figure out what makes the difference. So far the set up is very similar, with one exception: my project with incorrect metrics uses "exclusions" field. The script is added twice, but sends events from a subdirectory to a different Plausible project. I wonder if this may be the culprit? (btw it used to work fine in cloud Plausible)

**Reply by maximilianschmitt** [reply]
Interesting! I wasn't using the exclusions field in my project. By "the script is added twice" you mean that you've integrated the script two times on the same website? Have you tried removing the first script for the project with incorrect metrics and removing the "exclusions" field on that one?
